### DISCUSSION TITLE
Suggestions for docmost

### DISCUSSION THREAD

**Discussion by dante1024** [docmost/docmost#530] [Ideas]
Dear Project Author,
Hello!
I am a user of the open-source project you have on Github. First of all, I would like to express my sincere gratitude to you for creating and sharing this project. During my usage, I have found some areas that could potentially be optimized, and I would like to share my thoughts with you, hoping to contribute a little to the development of the project.
1. Support multiple language switching.
2. After logging in with other accounts, the user avatar and user picture in the upper right corner always show the administrator account. This should be a bug and I hope it can be fixed. I believe that after fixing this issue, the user experience will be smoother and more accurate, avoiding some unnecessary confusion.
3. currently, creating an account can only be done via email. I was wondering if it would be possible to add another way to create an account? For example, allowing users to directly use their personal names as account names. This would be more convenient and user-friendly for some users who don't want to disclose their emails or prefer a simpler account name.
4. I think it would be much more efficient in information management and retrieval if each page could be set with multiple tags and support searching pages by tags.
5. Support Markdown editing.
I fully understand the difficulty of developing a project, and you have already done an excellent job. I am just putting forward these ideas based on my own usage experience. Whether you have plans to update these aspects or not, I will always support this project. Looking forward to your reply and hoping the project will get better and better!
Thank you again for your hard work!

**Comment by LQ753799168** [comment]
Multilingual functionality is very important and we hope to support it soon

**Comment by WavedirectTel** [comment]
I agree with some of this sentiment.  I think this is a great project that should catch fire and hopefully more devs will jump in to help.  If I had one on staff I'd get them working on it.  

Language is not important to us, but it is globally for more people to use the product.  Markdown as an option is interesting but I think he's targeting a niche market to make things easier. Plenty of other wikis that support it.  

Tags yes

Also allowing a way of making some pages "public"

A file manager for smaller companies that don't use the cloud to upload docs and when you want to embed you can pick them from a list. 

One thing I LOVE that discord does is give a small summary of a link you post.  Not sure how you'd throw that function in.. but it would be super cool. 

For example 
![image](https://github.com/user-attachments/assets/d58d3229-ae1c-4d0b-b544-a2df22bb89df)
